deep_speech_2/deploy/README.md

```diff
@@ -0,0 +1,57 @@
+
```

Comment:
It seems that we don't need this, but please add related installations to setup.sh in the root dir.